frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarWhenExpression.scss

```diff
@@ -0,0 +1,23 @@
+.opp-task-sidebar-when-expression {
+  margin-bottom: var(--pf-global--spacer--lg);
+
+  &__section {
+    margin-bottom: var(--pf-global--spacer--md);
+    border: 1px dashed var(--pf-global--BorderColor--100);
+    padding: var(--pf-global--spacer--md);
+  }
+  &__control_button {
+    &_wrapper {
+      display: flex;
+      justify-content: flex-end;
+    }
```

Comment:
Small nit, try not to break elements in half with `&` it makes it harder to look them up.

Recommend just retyping the shared wording as it's the "element" name in BEM.


Double small nit, I think BEM wants `block-with-dashes__element-with-dashes`